### DISCUSSION TITLE
Custom color scheme for spatial plot does not seem to work

### DISCUSSION THREAD

**Discussion by smk5g5** [satijalab/seurat#4290] [Q&A] [answered]
I was trying to work with my spatial data and I am not able to make a custom coloring scheme work with spatial data? What am I doing wrong?

`SpatialPlot(object = scrna, features = i,images='B172.01',cols = c("grey88", "firebrick4"))` 

![image](https://user-images.githubusercontent.com/6811915/112552711-3e4dea00-8d91-11eb-9f34-fc1443a78510.png)

**Comment by timoast** [comment, marked as answer]
`cols` entry from the `SpatialPlot()` documentation:

> Vector of colors, each color corresponds to an identity class. This may also be a single character or numeric value corresponding to a palette as specified by brewer.pal.info. By default, ggplot2 assigns colors

So `cols = c("grey88", "firebrick4")` is not a valid parameter choice here for plotting a feature
